src/v3/src/components/Link/Link.tsx

```diff
@@ -63,38 +61,26 @@ const Link: UISchemaElementComponent<{
 
   return (
     typeof href === 'undefined' ? (
-      <LinkMui
-        component="button"
-        // Fixes OKTA-653788 (see comments) - Currently we treat all links as buttons
-        type="button"
-        variant="body1"
-        role="link"
+      <OdyLink
+        href=""
         onClick={onClick}
-        aria-describedby={ariaDescribedBy}
```

Comment:
Odyssey `Link` does not support `aria-describedby`. Confirmed that SIW does not currently set `ariaDescribedBy` at the transformer layer. If we add back this functionality in the future we can easily contribute to Odyssey and add support for `aria-describedby`